src/chapters/sql-part2/studio.rst

```diff
@@ -0,0 +1,59 @@
+Studio: Library
+===============
+
+Word has gotten out about your amazing SQL skills!
+The head librarian at the local library asked if you could help with their database.
+
+Database Setup
```

Comment:
Once the QA for the chapter content is done, we should place one or two internal links here for creating schemas and importing data.

Reply:
Sounds good!